institutions-api/src/main/resources/application.conf

```diff
@@ -8,6 +8,8 @@ hmda {
   institution {
     createSchema = false
     createSchema = ${?INSTITUTION_PG_CREATE_SCHEMA}
+    isBeta = false
```

Comment:
From a code perspective it seems like it would it would be easier to remove the `isBeta` flag and instead have different config maps in K8 for the default and beta tables and use the the helm charts to decide what tables get passed into the environment.

My thought is that the `hmda-analytics` service only ever has to know about one set of tables. So, we could move this into configuration and out of the code, specifically removing the additional repository classes that are included in this PR.